### DISCUSSION TITLE
Broken Symbolic Links

### DISCUSSION THREAD

**Discussion by mushie4282** [OpenVoiceOS/ovos-buildroot#249] [Q&A] [answered]
![image](https://github.com/OpenVoiceOS/ovos-buildroot/assets/106356342/38cf8c87-a5c1-46d0-9e84-6e3d62ade652)

Has anyone been experiencing symbolic link breakage to any ovos commands?

**Comment by goldyfruit** [comment]
Are you running the GUI?

**Reply by builderjer** [reply]
If the logs aren't there, the services didn't start. What image are you running?  Could be the message-bus because nothing will start without it.

**Reply by mushie4282** [reply]
No GUI
I'm running headless raspOVOS March 2 edition.

**Reply by builderjer** [reply]
Try the latest.  It is working well
https://ovosimages.ziggyai.online/raspbian/newest

**Comment by goldyfruit** [comment, marked as answer]
> No GUI I'm running headless raspOVOS March 2 edition.

Then its "normal" that these two symlinks are broken.
